ckanext/datastore_profiler/plugin.py

```diff
@@ -33,6 +33,7 @@ def get_actions(self):
         return {
             "update_profile": datastore_profiler.update_profile,
             "datastore_create": datastore_profiler.datastore_create_hook,
+            "error_display": datastore_profiler.error_display,
```

Comment:
Thanks for the initial change @reza-ghasemzadeh . However, we'll need to remove this line too. Not only is it misleading to have an action that doesnt exist, but Im worried CKAN might not start up with the extension like this

Reply:
I will remove it, too. Thanks for the note